.gitignore

```diff
@@ -22,8 +22,8 @@ rmm.egg-info/
 python/build
 python/*/build
 python/rmm/docs/_build
-python/rmm/**/librmm/**/*.cpp
-!python/rmm/librmm/_torch_allocator.cpp
+python/rmm/rmm/librmm/*.cpp
+!python/rmm/rmm/librmm/_torch_allocator.cpp
```

Comment:
It turns out that `rattler-build` _does_ respect `gitignore` negations, but the unignored locationw as incorrect, which is why the file wasn't included in the builds.